projects/cn0506/zcu102/system_project.tcl

```diff
@@ -0,0 +1,59 @@
+
+source ../../scripts/adi_env.tcl
+source $ad_hdl_dir/projects/scripts/adi_project_xilinx.tcl
+source $ad_hdl_dir/projects/scripts/adi_board.tcl
+
+##--------------------------------------------------------------
+# IMPORTANT: Set CN0506 interface mode
+#
+# The get_env_param procedure retrieves parameter value from the environment if exists,
+# other case returns the default value specified in its second parameter field.
+#
+#   How to use over-writable parameters from the environment:
+#
+#    e.g.
+#      make INTF_CFG="MII"
+#
+#    INTF_CFG  - Defines the interface type (MII, RGMII or RMII)
+#
+# LEGEND: MII
+#         RGMII
+#         RMII
+#
+##--------------------------------------------------------------
+
+if {[info exists ::env(INTF_CFG)]} {
+  set INTF_CFG [get_env_param INTF_CFG "MII"]
+} elseif {![info exists INTF_CFG]} {
+  set INTF_CFG "MII"
```

Comment:
I think that RGMII should be the default, since it has the highest available rates.
In system_bd.tcl you are extracting the value from the INTF_CFG in the environment. There will be a problem if you want to use the default interface(no configuration defined) because the system_bd.tcl does not get the default value set above. So, you should add another command which sets and environment variable. See: https://github.com/analogdevicesinc/hdl/blob/master/projects/dac_fmc_ebz/common/config.tcl#L13

This should be done for all carriers.